ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -121,6 +122,25 @@ const triggerSet: TriggerSet<Data> = {
       diveFromGraceHasArrow: { 1: false, 2: false, 3: false },
     };
   },
+  timelineTriggers: [
+    {
+      id: 'DSR Resentment',
+      regex: /Resentment/,
+      beforeSeconds: 5.7,
```

Comment:
I used 5.7 as that was the cast time of the Earthen Fury spell from e4n where cactbot uses startsUsing as the trigger.

Reply:
Sorry, I'm not sure I see what you're referring to.  It sounds unrelated though?  Would you mind picking a more round number like 5 (long enough for mit, short enough for tank personals before autos if desired).

Reply:
I am referencing this trigger: https://github.com/quisquous/cactbot/blob/b61c05e3b7e045d8aeda025b8d5d34a32ac78455/ui/raidboss/data/05-shb/raid/e4n.ts#L20
```
{
      id: 'E4N Earthen Fury',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: '40F8', source: 'Titan', capture: false }),

      infoText: (_data, _matches, output) => output.text!(),
      outputStrings: {
        text: {
          en: 'aoe + dot',
          de: 'AoE + DoT',
          fr: 'AoE + dot',
          ja: 'AoE + DoT',
          cn: 'AOE + dot',
          ko: '전체공격 + 도트뎀',
        },
      },
```

Reply:
Although looking at the spells there more closely, it's not the 40F8 that applies the hit, but the longer spell with 7.20s that does, so it's more of a 7.3s or so warning in that case. The other reference I have is from Delubrum Reginae Savage where we chose 8s, but there I think is a stronger aoe + bleed and somewhat different scenario as here we are more likely to recover.

Reply:
> I am referencing this trigger:

Oh! Sorry, I was looking for a literal `5.7` somewhere in the codebase and didn't see it.  (Also, I think cast times are 0.3 less than what they actually are, so that's 6, unless I'm mistaken?)

Yeah, I think 5 or 8 are both reasonable.  I just don't think there's a call for something so specific as `5.7` here, as somebody in the future will copy this and then be like "why is this 5.7??".